cypress/integration/jobs-view-test.js

```diff
@@ -1,49 +1,51 @@
 describe('Jobs View', () => {
 
   beforeEach(() => {
-    // cy.fixture('../fixtures/jobData.json')
-    // .then((jobs) => {
-    //   cy.intercept('http://localhost:3000/')
-    // })
-    cy.visit('http://localhost:3000/')
+    cy.stubAllJobs()
   });
 
-  describe('Page Load View', () => {
+  describe('Page Load User Story', () => {
 
     it('Should display the company name on load', () => {
       cy.get('h1').contains('Space Truckers')
     });
 
     it('Should display all jobs on load', () => {
       cy.get('.job-container')
-        .find('.card').should('have.length', 4)
+        .find('.card').should('have.length', 2)
     });
 
     it('Should load individual job data into each Card', () => {
       cy.get('.card').get('#1')
-        .get('h3').contains('Mars')
+        .get('h2').contains('Mars')
         .get('.date').contains('11:10:2195')
         .get('.pay').contains(50000000)
-        .get('.fluff').contains('Mars is a barren planet with little resources after it was used up by hyper capitalists in the early 21st century.')
+        .get('#book1').contains('Book Job')
+        .get('.fluff').contains('Mars is a barren planet with little resources.')
     });
   });
 
-  describe('Job Booking', () => {
+  describe('Delete Booking User Story', () => {
 
-    it('Should change the job value to booked when the booked button is clicked', () => {
-      cy.get('#book2')
+    it('Should delete a job when the delete button is clicked', () => {
+      cy.removeJob()
+      cy.get('#delete2')
         .click()
-        .get('.card').get('#2')
-        .get('.booked').contains('Job booked')
+        .get('.card').should('have.length', 1)
+      });
     });
-  });
 
-  describe('Delete Booking', () => {
+  describe('Job Booking User Story', () => {
 
-    it('Should delete a job when the delete button is clicked', () => {
-    cy.get('#delete3')
-      .click()
-      .get('.card').should('have.length', 3)
+    it('Should change the job value to booked when the booked button is clicked', () => {
+      cy.stubAllJobs()
```

Comment:
I think this line is unnecessary since there is a before each in which we call stubAllJobs

Reply:
Added change